openlane_update_tool/action.yml

```diff
@@ -0,0 +1,40 @@
+name: Update OpenLane Tool
+
+inputs:
+  tool_name:
+    description: Name of the tool to update.
+    required: true
+  pdk_root:
+    description: Path to the PDK folder.
+    default: .
+  build_ol:
+    description: Build docker image after tool update.
+    default: true
```

Comment:
what happens if user don't set this to true, does `openlane_run` end up using the updated version?

Reply:
If the user wants to update multiple tools in one workflow they can skip the build on all but the last update. Added note about the usage on the docs.

Reply:
so in all cases the last `openlane_update_tool` of any workflow will always needs to have `build_ol=true` for `openlane_run` to benefit from it?

did you consider taking a list of tools to update instead? (github action don't support list but they apparently support multiline strings)